### ISSUE TITLE
Specify arenas via config

### ISSUE THREAD

**Issue by Eradiator** [Tofpu/bedwars-swap-addon#4]
**_REQUEST_**
Please add more customisation to add more custom arena groups.
which can help in organising the games into Solo, Doubles,  3v3v3v3,  4v4v4v4,  etc
**_Example_**
/bw arenaGroup create Solo-swap
/bw arenaGroup create Doubles-swap
**_CONFIGURATION_**
general-settings:
    enable-debug-mode: true
swap-announce-settings:
    minimum-swap-interval: 60
    maximum-swap-interval: 180
**swap-group:
    - Solo-swap
    - Doubles-swap**

**Comment by KKiiya** [comment]
Alr the developer seems to be inactive so ill work on this instead

**Comment by Tofpu** [comment]
> Alr the developer seems to be inactive so ill work on this instead

Yeah... unfortunately I have been very busy lately... but I do feel obligated to tell you that the project is currently in a middle of a rewrite. If you would like, I could push the latest code to a new branch: but be aware that it's slightly broken.

**Comment by Tofpu** [comment]
> I see, I hope you are doing good

I am indeed doing well, thank you for asking! I hope you are too!

> Yes, I would like that, thanks

Alright, I will try to push the latest code of the rewrite within a couple of hours. 

Side note, I have been recently planning to reallocate some of my time spent on Speedbridge to my other projects. And after some deep consideration, I think perhaps I could (justifiably) allocate some time for swappage. Mainly to eliminiate the irritating bugs that currently roam the project (#2 for instance).

If you would like to stay update-to-date with swappage: do consider joining the [discord server](https://discord.com/invite/saMvytm) (if you haven't already)!
